### PR TITLE
Updated UserGuide.adoc and AboutUs.adoc

### PR DESCRIPTION
**Pull request by kensonoenqh**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -135,9 +135,23 @@ Expected output:
...
+Format: `list`
+
+Examples:
+
+*  `list`
+*  `list m/Jan t/Food`
+
+Expected output:
+
+* Returns a list of all expenses for the current month and remaining budget left.
```
Comment (limerencee) [inline docs/UserGuide.adoc]: I think remaining budget will be shown on the Ui at all times, so maybe do not have to state it here which might lead to confusion.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -135,9 +135,23 @@ Expected output:
...
+* Returns a list of all food expenses for the specified month and remaining budget left.
```
Comment (limerencee) [inline docs/UserGuide.adoc]: Same as the line above

**Review by limerencee — CHANGES_REQUESTED** [review]
Please modify the wording slightly!

**Comment by krusagiz** [comment]
I would suggest merging this PR before @lye-jw's as it is shorter. You can choose to merge this later, if the requested changes are not done quickly.

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -135,9 +135,23 @@ Expected output:
 
 * Adds the expense spent on Laksa back to THRIFT.
 
+=== List expenses for the month: `list`
+
+Format: `list`
```
Comment (limerencee) [inline docs/UserGuide.adoc]: Should be:
Format: \`list [m/MONTH]\ [t/TAG]`

**Review by limerencee — CHANGES_REQUESTED** [review]

**Review by limerencee — APPROVED** [review]

**Pull request merged**